### PR TITLE
Add appveyor configuration file

### PR DESCRIPTION
**Pull request by DaRamirezSoto**
Added a configuration file for AppVeyor tried to mimic how it was defined in travis configuration file.
This is first time for me using AppVeyor so it defiantly need some more experienced eyes.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - x64
+
+install:
+  - ps: Install-Product node $env:nodejs_version $env:platform
+  - npm install
+
+test_script:
+  - node --version
+  - npm --version
+  - IF %nodejs_version% EQU 7 npm run travis:lint
```
Comment (stof) [inline appveyor.yml]: linting does not need to be done on both Travis and Appveyor. It only depends on the source code, which is the same. I would remove it from here

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Should we cache local modules ?
+# cache:
+#  - node_modules
+
+environment:
+  matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "7"
+    - nodejs_version: "8"
```
Comment (stof) [inline appveyor.yml]: Do we actually need to test all node versions on Appveoyr as well ? I don't think it is worth it. AFAIK, Appveoyr is not as good as Travis at handling multiple jobs (especially regarding cache, even though you don't use it yet).

Comment (DaRamirezSoto) [inline appveyor.yml]: @stof I agree, I just followed the specs as given by `package.json` which states >= 6.
I can change to only test on 6 if that is what we desire

File: appveyor.yml (hunk 1 above)
Comment (DaRamirezSoto) [inline appveyor.yml]: @stof I just followed the configuration that was given in travis config file.
If we decide on not including this I can remove it

**Comment by fabpot** [comment]
@weaverryan I've just enabled Appveyor on symfony/webpack-encore

**Comment by weaverryan** [comment]
@DaRamirezSoto Awesome! The build is running great - it looks like it's the one failure you mentioned over and over again. I'll merge so that I can open a PR to get rid of that :). Thanks!

@fabpot I tried to enabled it yesterday through my account, but it gave the project a `weaverryan/webpack-remix` name. And I still don't see https://ci.appveyor.com/project/symfony/webpack-encore after you've enabled it.

**Comment by fabpot** [comment]
@weaverryan hehe, AppVeyor works very differently from Travis. So, the URL is https://ci.appveyor.com/project/fabpot/webpack-encore

**Pull request merged**